MobiMart/templates/base.html

```diff
@@ -1,3 +1,6 @@
+<!-- Title bar icon -->
+    <link rel = "icon" href ="./static/images/Logo.png" "type "image/x-icon"> 

```

Comment:
shouldn't these tag be added in title section?

Reply:
Yes, I added these tags in the title section first but when I opened base.html in chrome icon was not updated.
So I added these tags above {% load static %}.

Now, title icon is updated.

Reply:
Yes, you were right.

So, should I create a new pull request correcting all these errors and close this one ?